tests/stacks/aws/test_data/expected/apps_v1_deployment_centraldashboard.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     app: centraldashboard
     app.kubernetes.io/component: centraldashboard
     app.kubernetes.io/name: centraldashboard
```

Comment:
Looking at your PR, this label shouldn't exist now. Is this a mistake?

Reply:
That will be coming from the `kustomization.yaml` in `overlay/application`:  https://github.com/kubeflow/manifests/blob/master/common/centraldashboard/overlays/application/kustomization.yaml

Reply:
Sorry, still not fully clear.
Why do some test snapshots have the two labels, while some do not?
Is that expected?

Reply:
If the stack is using the `application` overlay, there will be both, but not otherwise (because I removed them from the base).

Ones from `application` overlay: 
```yaml
    app.kubernetes.io/component: centraldashboard
    app.kubernetes.io/name: centraldashboard
```

Ones from base: 
```yaml
    app: centraldashboard
    kustomize.component: centraldashboard
```

Reply:
Thanks for clarification! Created an issue to track the problem: https://github.com/kubeflow/manifests/issues/1649.